<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <pointsize>10</pointsize>
   </font>
  </property>
  <property name="windowTitle">
   <string>CommunicationBoard</string>
  </property>
  <property name="windowIcon">
   <iconset>
    <normaloff>../Resources/icon.png</normaloff>../Resources/icon.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QComboBox" name="ProfileComboBox">
    <property name="geometry">
     <rect>
      <x>260</x>
      <y>150</y>
      <width>280</width>
      <height>60</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>-1</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true">QComboBox{
	border: 4px solid #ced4da;
	border-radius: 4px;
	font-size: 16px;
}

QComboBox::drop-down {
	border: 0px;
}

QComboBox::down-arrow {
    image: url(./Resources/arrow-down.svg);
}

QComboBox:on { /* shift the text when the popup opens */
   border: 4px solid #c2dbfe;
}

QComboBox::QListView{
	font-size: 16px;
	border: 4px solid rgba(0, 0, 0, 10%);
	background-color: #fff;
	outline: 0px;
}

QComboBox::QListView::item{
	background-color: #fff;
}

QComboBox::QListView::item::hover{
	background-color: #1e90ff;
}
QComboBox::QListView::item::selected{
	background-color: #1e90ff;
}




</string>
    </property>
    <property name="editable">
     <bool>true</bool>
    </property>
    <property name="currentText">
     <string>בחר פרופיל</string>
    </property>
    <property name="currentIndex">
     <number>-1</number>
    </property>
    <property name="maxVisibleItems">
     <number>10</number>
    </property>
   </widget>
   <widget class="QPushButton" name="AddProfileButton">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>440</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>הוסף פרופיל</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Resources/UI Icons/plus-circle.svg</normaloff>../Resources/UI Icons/plus-circle.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="DeleteProfileButton">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>440</y>
      <width>140</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>הסר פרופיל</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Resources/UI Icons/minus-circle.svg</normaloff>../Resources/UI Icons/minus-circle.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLineEdit" name="ProfileLineEdit">
    <property name="geometry">
     <rect>
      <x>230</x>
      <y>390</y>
      <width>341</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
    <property name="placeholderText">
     <string>הכנס שם פרופיל שברצונך להוסיף/להסיר</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>40</y>
      <width>230</width>
      <height>50</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>20</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ברוכים הבאים</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QLabel" name="ErrorLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>500</y>
      <width>781</width>
      <height>20</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>11</pointsize>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="LoadBoardButton">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>230</y>
      <width>160</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>טען לוח תקשורת</string>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>../Resources/UI Icons/loader.svg</normaloff>../Resources/UI Icons/loader.svg</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>20</width>
      <height>20</height>
     </size>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>285</x>
      <y>330</y>
      <width>230</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>ניהול פרופילים</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
